### ISSUE TITLE
Jackal Shields Block Flanks

### ISSUE THREAD

**Issue by ghost** [HaloSpaceStation/HaloSpaceStation13#2731]
(noted as a bug and not intended by XO)
#### Description of issue
When firing at a jackal directly from the side of the shield's facing (i.e. the shield is facing east, but you are firing from direct north) the shield will still block the shot, observed during a few rounds today and yesterday. 

#### Difference between expected and actual behavior
The shield should be blocking its facing & diagonals, the sides of the shield should be unprotected and vulnerable to being flanked

#### Steps to reproduce 
have a (standing) jackal with its shield up, face a direction. Shoot from the direct sides of the shield (i.e. if it faces south/north, shoot from east/west and vice versa)

#### Length of time in which bug has been known to occur
Probably indefinite

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [X] Issue could be reproduced in multiple rounds
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/HaloSpaceStation/HaloSpaceStation13/issues)

**Comment by ondrej008** [comment]
not indefinite, i've seen jackals die from the sides, this seems rather new

**Comment by X0-11** [comment]
jackal shields haven't been changed iirc

**Comment by ghost** [comment]
Yeah they haven't seen any changes for a long while, they needed a bugfix like two years ago where they weren't properly blocking shots and I don't think they've been touched since

**Comment by Sgtwok** [comment]
I haven't seen this, and I easily play kig-yar the most.

**Comment by X0-11** [comment]
confirmed with live test

**Comment by caelaislinn** [comment]
XO is incorrect, this was intended. When I redid shields a month ago I increased the shield coverage angle to 3/4 (intending it to be 270 degrees). However due to Byond directional snapping, which I used as a shortcut, the coverage angle in degrees is actually closer to about 315 degrees. 

However I jumped on the git history and noticed XO put in a change to shields a few days ago (5/8/2020) which may be actually caused a 100% coverage bug 😃

**Comment by Sgtwok** [comment]
Kig-yar are meant be be flanked but a 270 degree angle is a little too forgiving for kig-yar, personally i'm favor of the old system which makes kig-yar difficult to take on from the front but able to be flanked without having to go essentially behind the kig-yar.

And, coming from me, must surely have some weight.

**Comment by loafe** [comment]
I have come from my banishing in gay baby jail just to say that your opinion is just as worthless as mine matt

change it back to 180 degrees like god intended

**Comment by Aroliacue** [comment]
Fixed

**Comment by YourPalHolidays** [comment]
Closing.
